### PR TITLE
Enable profiling

### PR DESCRIPTION
**Pull request by sonyafenge**
**What type of PR is this?**
> /kind bug
> /kind feature

**What this PR does / why we need it**:

- Enable profiling for kube-scheduler and kube-controllermanager by default: This commit refer to community PR:

https://github.com/kubernetes/kubernetes/pull/84835
https://github.com/kubernetes/kubernetes/pull/88663

- Using ETCD in-secure ports: 2382 to collecting pprof and prometheus debug data 

**Which issue(s) this PR fixes**:

Fixes #https://github.com/CentaurusInfra/arktos/issues/1008

**Special notes for your reviewer**:
Perf-test with 100 hollow-nodes finished with success for both density and load. Detailed log can be found under GCP project: workload-controller-manager on sonya-uswest2: /home/sonyali/logs/perf-test/gce-100/arktos/profiling-0310-100-1tp/arktos
**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cmd/controller-manager/app/options/debugging.go [hunk 1]
```diff
@@ -27,6 +28,16 @@ type DebuggingOptions struct {
 	*componentbaseconfig.DebuggingConfiguration
 }
 
+// RecommendedDebuggingOptions returns the currently recommended debugging options.  These are subject to change
```
Comment (yb01) [inline cmd/controller-manager/app/options/debugging.go:31]: to clarify: there is a commandline args to set this option for controllers, any particular reason this logic is added here to control it ?

Comment (sonyafenge) [reply cmd/controller-manager/app/options/debugging.go:31]: This is refer to community PR: https://github.com/kubernetes/kubernetes/pull/88663

**Review by yb01 — APPROVED** [review]
lgtm.
discussed with Sonya. we will take the community's approach to get the profiling enabled as for it default value.

**Pull request merged**